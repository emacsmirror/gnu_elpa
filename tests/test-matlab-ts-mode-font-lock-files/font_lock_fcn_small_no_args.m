% -*- mode: matlab-ts -*-
function font_lock_fcn_small_no_args
% help comment

    % code comment
    disp('no args');
end
