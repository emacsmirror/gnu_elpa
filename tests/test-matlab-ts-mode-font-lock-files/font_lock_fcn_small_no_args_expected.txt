c ccc ccccc ccccccccc ccc
kkkkkkkk fffffffffffffffffffffffffff
h hhhh hhhhhhh

    c cccc ccccccc
    BBBBbSss ssssSbD
kkk
